README.md

```diff
@@ -0,0 +1,118 @@
+# README
```

Comment:
- [ ]  Your API does not have any documentation. Please add documentation about how to consume your API:

![https://user-images.githubusercontent.com/57421823/200048389-b9654251-91e3-4642-9cd5-c65f71a11960.png](https://user-images.githubusercontent.com/57421823/200048389-b9654251-91e3-4642-9cd5-c65f71a11960.png)

If your team has 3 or more people, kindly add the required documentation for the API.

You could use [rswag](https://github.com/rswag/rswag) which will add the documentation and tests at the same time 💪